openfecwebapp/api_caller.py

```diff
@@ -268,3 +259,47 @@ def load_top_parties(sort, cycle=2016, per_page=5):
         if response['results']:
             return response
         return {}
+
+def _get_sorted_respondents(mur):
+    """
+    Returns the respondents in a MUR sorted in the order of most important to least important
+    """
+    SORTED_RESPONDENT_ROLES = ['Primary Respondent', 'Respondent', 'Previous Respondent']
+    respondents = []
+    for role in SORTED_RESPONDENT_ROLES:
+        respondents.extend(sorted([p['name'] for p in mur['participants'] if p['role'] == role]))
+    return respondents
+
+def _get_sorted_participants_by_type(mur):
+    """
+    Returns the participants in a MUR sorted in the order of most important to least important
+    """
+    SORTED_PARTICIPANT_ROLES = [
+        "Primary Respondent",
+        "Respondent",
+        "Previous Respondent",
+        "Treasurer",
+        "Previous Treasurer",
+        "Complainant",
+        "Respondent's Counsel",
+        "Opposing counsel",
+        "Representative",
+        "Law Firm",
+    ]
```

Comment:
@vrajmohan I just checked with @emileighoutlaw to be sure on this, and she confirmed: Could we adjust the capitalization of these fields to have only the first word capitalized?
`Primary respondent` 
`Previous respondent` 
`Previous treasurer`
`Respondent's counsel`
`Law firm`

We realize the Document tables in MURs will need this as well, and @emileighoutlaw is going to file a separate issue for that for the next sprint.